opencraft/settings.py

```diff
@@ -77,7 +77,8 @@
     'pr_watch',
     'userprofile',
     'registration',
-    'backup_swift'
+    'backup_swift',
+    'community',
```

Comment:
@bradenmacdonald Sorry for seeing that only now - I wasn't pinged explicitly so that thread went unnoticed for some time.

> How would people get the results though? Are you saying we should commit the output into a .yml file in the test suite, and then add a new test that generates the output and compares it to the saved output? (Then we can link to that .yml file?) 

Yup.

> That would work, except for the fact that the current approach has randomly generated secure strings for all the secret variables, and we couldn't include that feature if doing it as part of the test suite.

This could be mocked up, no?